.github/workflows/temp_release.yml

```diff
@@ -0,0 +1,59 @@
+name: Temp Release Workflow
```

Comment:
We can also run these steps manually vs creating a removing a new workflow. That's what we've typically done previously on iOS at least unless some of these steps cannot be done manually.

Reply:
We don't currently have that option with how the workflow is set up. Right now each job requires the previous one: https://github.com/braintree/braintree_android/blob/e5500e0f270b7fd14e66eee19ec461c262ad1cef/.github/workflows/release.yml#L249

I feel that there's a benefit in restricting the running of individual jobs and enforcing the entire workflow to be run for a release. I worry that it'd open up the possibility of publishing something to maven central by accident.

Reply:
iOS does not rely on the same settings, really the [iOS releases](https://github.com/braintree/braintree_ios/blob/main/.github/workflows/release.yml) as a whole are pretty different than what we do on android (for example we don't run all the tests again). It is nice in that say cocoapods or publishing reference docs fails we typically just run those 1-2 steps manually. We do rely on the order of things but can pick up where the release failed if something goes sideways.

Reply:
Gotcha. I think the Android release workflow has been stable up until v5, so hopefully we won't have any of these CI issues until the next major version where our gradle version and build tools are bumped.

Are you okay with merging in the temp workflow, then following up with another PR to remove it?